src/flat-world.js

```diff
@@ -101,7 +101,8 @@ const flatWorld = module.exports = {
   },
 
   getPixelRatio: function getPixelRatio(ratio) {
```

Comment:
the pixelratio fixed I introduced before supports retina rendering, but breaks picking on retina. so I temporally disabled it. It would be great to have someone look into this, and have the pixelratio working with picking as well